client/app/scripts/liveblog-edit/module.js

```diff
@@ -480,18 +480,17 @@ define([
         function canPublishAPost(blog) {
             return privileges.userHasPrivileges({'publish_post': 1});
         }
-        function isUserOwner(archive) {
-            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
-                return true;
-            } else {
-                return false;
-            }
+        function isAdmin() {
+            return $rootScope.currentUser.user_type === 'administrator';
+        }
+        function isUserOwnerOrAdmin(archive) {
+            return $rootScope.currentUser._id === archive.original_creator || isAdmin();
         }
         function goToSettings() {
             var def = $q.defer();
             blogService.get($route.current.params._id)
             .then(function(response) {
-                if (isUserOwner(response)) {
+                if (isUserOwnerOrAdmin(response)) {
```

Comment:
As you can see here https://github.com/superdesk/liveblog/pull/167
An owner or an admin can do different things. How do you plan to identify these rights ?
The technical specifications describe clearly all the roles. What can change in the future ?
https://twitter.com/dhh/status/623598101127897088